portlist/portlist.go

```diff
@@ -18,6 +18,7 @@ type Port struct {
 	Proto   string // "tcp" or "udp"
 	Port    uint16 // port number
 	Process string // optional process name, if found
+	Pid     int    // process id, if known
```

Comment:
Thought you were planning on adding a bool for localhost on ports? Guess not needed?

Reply:
@bradfitz yeah it's not needed for now since we changed how we do the filtering, but I'm happy to add it back in a follow up PR.